extensions/zoom-control/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "zoom-meeting-control",
+  "title": "Zoom Meeting Control",
+  "description": "Control current Zoom meeting",
+  "icon": "zoom-control.png",
+  "author": "prashantv",
+  "contributors": [
+    "pernielsentikaer"
+  ],
+  "categories": [
+    "Communication"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "icon": "zoom-mute.png",
+      "name": "mute",
+      "title": "Zoom - Mute Meeting",
```

Comment:
Should the `Zoom - ` part be removed from the command? The extension guidelines suggest it just being a `<verb> <noun>`, without a prefix

Reply:
It should! I already changed that [here](https://github.com/raycast/extensions/pull/8817/commits/a6b96a0aeafbb70f0af41a9365cce860f9f11803) but it seems like it got pushed back 🙂